src/index.js

```diff
@@ -88,9 +75,11 @@ const getFirstReplacementValueForNode = (obj, comparator, nodePath) => {
  * @return {undefined}
  */
 const processNode = (replacements, nodePath, replaceFn, comparator) => { // eslint-disable-line
-  const replacement = getFirstReplacementValueForNode(replacements, comparator, nodePath);
-  if (replacement) {
-    replaceAndEvaluateNode(replaceFn, nodePath, replacement);
+  const replacementKey = getSortedObjectPaths(replacements)
```

Comment:
Humm, this replaces a named function with docs with inline code. I think that I prefer to keep the named function  but I can see how this is more succinct which also has merit. If you feel strongly about this change maybe you could update the doc on `processNode` to say something more descriptive than `Run the transformation over a node` since this change would remove some of the context.